Navitia.php

```diff
@@ -2,173 +2,78 @@
 
 namespace CanalTP\NavitiaPhp;
 
-use CanalTP\NavitiaPhp\Api\Coords;
-use CanalTP\NavitiaPhp\Api\Lines;
-use Symfony\Component\Yaml\Yaml;
-use Symfony\Component\OptionsResolver\OptionsResolver;
 use GuzzleHttp\Client;
 use GuzzleHttp\Exception\ClientException;
-use GuzzleHttp\Exception\RequestException;
-use GuzzleHttp\Exception\ServerException;
 use CanalTP\NavitiaPhp\Exception\NavitiaException;
-use CanalTP\NavitiaPhp\Api\Coverage;
-use CanalTP\NavitiaPhp\Api\TrafficReports;
 
 class Navitia
 {
-    private $httpClient;
     /**
-     * @var string
+     * @var Client
      */
-    private $defaultCoverage;
```

Comment:
Yes we can provide a "default" coverage:

``` php
$queryBuilder->coverage('fr-idf');
```

Then the query builder have a coverage that can be overrided.
The configuration of a default coverage for Symfony could be done in service declaration